src/ResourceManagement/Cdn/Microsoft.Azure.Management.Cdn/Generated/Models/EdgeNode.cs

```diff
@@ -29,36 +32,38 @@ public partial class EdgeNode
         /// <summary>
         /// Initializes a new instance of the EdgeNode class.
         /// </summary>
-        /// <param name="name">Ip adress group that contains Ipv4 and Ipv6
-        /// addresses</param>
-        /// <param name="resourceGroup">The resource group of the edge
-        /// node.</param>
+        /// <param name="location">Resource location.</param>
         /// <param name="ipAddressGroups">List of ip address groups.</param>
-        public EdgeNode(string name = default(string), string resourceGroup = default(string), IList<IpAddressGroup> ipAddressGroups = default(IList<IpAddressGroup>))
+        /// <param name="id">Resource ID.</param>
+        /// <param name="name">Resource name.</param>
+        /// <param name="type">Resource type.</param>
+        /// <param name="tags">Resource tags.</param>
+        public EdgeNode(string location, IList<IpAddressGroup> ipAddressGroups, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>))
```

Comment:
@csmengwan changing the order of parameters in the constructor is a breaking change